tests/test_db_service.py

```diff
@@ -96,11 +97,25 @@ async def test_delete_deletes_data(self):
         self.collection.delete_one.return_value = futurized(
             DeleteResult({}, True)
         )
-        success = await self.test_service.delete("yolo", self.id_stub)
+        success = await self.test_service.delete("test", self.id_stub)
         self.collection.delete_one.assert_called_once_with({"accessionId":
                                                            self.id_stub})
         assert success
 
+    def test_query_executes_find(self):
+        """Test that find is executed, so cursor is returned."""
+        self.collection.find.return_value = AsyncIOMotorCursor(None, None)
+        cursor = self.test_service.query("test", {})
+        assert type(cursor) == AsyncIOMotorCursor
```

Comment:
maybe `self.assertEquals` ? using an `AsyncTestCase` also does not seem to be the only place. look at https://docs.python.org/3.7/library/unittest.html for a complete list of what could be used.

Reply:
I'm not following - how does `AsyncTestCase` relate to `assertEquals` here?

Also, why assertEquals instead of assert? I think assert makes code more readable.

Reply:
it is not about readable code it is about using the `unittest.Testcase` class methods which are inherited: https://github.com/kwarunek/aiounittest/blob/master/aiounittest/case.py#L6 
`assert` has a place but it is not in a unit test class

Reply:
https://stackoverflow.com/questions/48216473/using-asserttrue-vs-assertequal-in-unittest

Reply:
Fair point. But what do you mean with this: "using an AsyncTestCase also does not seem to be the only place"?

Reply:
missing punctuation my bad. I meant that this is not the only place you used `assert` instead of `assertEquals` or similar
comment updated.

Reply:
Ah okay. Ok if I make another PR to change `assert`s?

Reply:
well ok .. I will review the rest of the PR tomorrow.

Reply:
Wasn't actually that big of a deal, so I included assert fixes in this PR